### PR TITLE
Re-enable airdrop labels

### PR DESCRIPTION
**Pull request by aalan3**
This PR fixes https://github.com/duneanalytics/spellbook/pull/4540

### CODE REVIEW

File: macros/dune/source.sql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% macro source(source_name, table_name) %}
   {% set rel = builtins.source(source_name, table_name) %}
-  {%- if target.type == 'trino' -%}
+  {%- if target.type == 'trino' and source_name != 'dune' -%}
```
Comment (jeff-dude) [inline macros/dune/source.sql:3]: i guess this macro change breaks the legacy check step?

Comment (aalan3) [reply macros/dune/source.sql]: I think everything becomes `modified` which then triggers the legacy check for all unmigrated models?

**Comment by jeff-dude** [comment]
i believe this is the PR that @couralex6 mentioned he could take a look at, in terms of changing state modified to only body in the CI test so that the macro doesn't trigger everything?

we promised alan notes, so just wanted to follow up here, even if we don't make any progress since we're focused on finalizing migrating spells

**Comment by jeff-dude** [comment]
closing for now, until we find need for resolving dune file uploads

**Pull request closed without merging**